lib/get-vrt-settings.js

```diff
@@ -10,6 +10,15 @@ const get = require('lodash/get')
  * @return {NightwatchVRTSettings} An object containing the visual regression test settings.
  */
 module.exports = function getVrtSettings(nightwatchClient, overrideSettings) {
+    const visualRegressionSettingsPaths = [
+        'globals.test_settings.visual_regression_settings',
+        'globals.visual_regression_settings'
```

Comment:
Based on the logic to resolve settings below, it seems like this order should be reversed so that `globals.visual_regression_settings` takes precedence if both are defined (for whatever reason).